sources/src/main/java/com/google/solutions/jitaccess/web/RuntimeConfiguration.java

```diff
@@ -68,6 +68,9 @@ public RuntimeConfiguration(Function<String, String> readSetting) {
     this.maxNumberOfJitRolesPerSelfApproval = new IntSetting(
       List.of("ACTIVATION_REQUEST_MAX_ROLES"),
       10);
+    this.listAllAvailableProjectsIn = new StringSetting(
+      List.of("LIST_ALL_AVAILABLE_PROJECTS_IN"),
```

Comment:
If we switch to `search`, then this would need to become a boolean setting (`LIST_ALL_AVAILABLE_PROJECTS` or so)

Reply:
I think `search` is even more powerful than `list` since they support queries, (e.g. using folders, labels, etc), so I just moved from parent field to query. What do you think?

Reply:
Yes, I think it's fine to let users configure the query.

However, the variable name  `SEARCH_ALL_AVAILABLE_PROJECTS` sounds like a boolean. Could we rename that to 
`AVAILABLE_PROJECTS_QUERY` or so?

The rest looks good, thanks!

Reply:
Initially, it was a boolean, so I guess the soul survived in the naming :) Totally make sense, renamed it to `AVAILABLE_PROJECTS_QUERY`